src/components/common/verify-code/verify-code-controller.ts

```diff
@@ -63,6 +64,14 @@ export function verifyCodePost(
         return renderBadRequest(res, req, options.template, error);
       }
 
+      if (supportReauthentication() && req.session.user.reauthenticate) {
+        if (result.data.code === ERROR_CODES.ENTERED_INVALID_MFA_MAX_TIMES) {
```

Comment:
Because you don't have any "else" for this "if", that means you can add this condition to your first `if`.
That will simplify your check.